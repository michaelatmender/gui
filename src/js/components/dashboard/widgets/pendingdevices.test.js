import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { createMount } from '@material-ui/core/test-utils';
import PendingDevices from './pendingdevices';

it('renders correctly', () => {
  const tree = createMount()(
    <MemoryRouter>
      <PendingDevices />
    </MemoryRouter>
  );
  expect(tree.html()).toMatchSnapshot();
});
