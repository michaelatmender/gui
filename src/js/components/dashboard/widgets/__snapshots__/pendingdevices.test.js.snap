// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `"<div style=\\"position: relative;\\"><a class=\\"MuiButtonBase-root MuiFab-root MuiFab-primary\\" tabindex=\\"0\\" role=\\"button\\" aria-disabled=\\"false\\" style=\\"position: absolute; top: -28px; left: 15px; z-index: 1;\\" href=\\"/devices/pending\\"><span class=\\"MuiFab-label\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\" role=\\"presentation\\"><path d=\\"M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z\\"></path></svg></span><span class=\\"MuiTouchRipple-root\\"></span></a><div class=\\"notActive widget \\"><div style=\\"height: 100%; width: 100%; display: flex; flex-direction: column;\\"><div style=\\"display: flex; flex-direction: column; align-items: flex-end;\\" class=\\"widgetMainContent align-right\\"><div class=\\"header\\">Pending devices</div><div class=\\"counter\\"></div></div><span class=\\"link\\">View details</span></div></div></div>"`;
